saleor/graphql/account/bulk_mutations.py

```diff
@@ -70,6 +73,12 @@ def perform_mutation(cls, _root, info, ids, **data):
     @classmethod
     def clean_instances(cls, info, users):
         errors = defaultdict(list)
+
+        if bool(getattr(info.context, "app", None)):
```

Comment:
AFAIK you should be able to simply make `info.context.app`. `app` is always there.